File: MAB.Domain/Servico.cs
using System;

namespace MAB.Domain
{
    public class Servico
    {
        public int Id { get; set; }
        public string Defeito { get; set; }
        public string ServicosExecutados { get; set; }
        public string PecasSubstituidas { get; set; }
        public string Observacao { get; set; }
        public DateTime DtAtendimento { get; set; }
        public decimal ValorServico { get; set; }
        public bool Status { get; set; }
    }
}

File: SGCOS-App/src/app/_models/Servico.ts
export interface Servico {
    id: number;
    defeito: string;
    servicosExecutados: string;
    pecasSubstituidas: string;
    observacao: string;
    dtAtendimento: Date;
    valorServico: number;
    qtdDiasGarantia: number;
    equipamentoId: number;
}
